src/status/libstatus/wallet.nim

```diff
@@ -58,14 +61,42 @@ proc getTransfersByAddress*(address: string): seq[Transaction] =
     error "Failed getting wallet account transactions", msg
     
 
-proc sendTransaction*(from_address: string, to: string, value: string, password: string): string =
-  var args = %* {
-    "value": fmt"0x{toHex(value)}",
-    "from": from_address,
-    "to": to
-  }
-  var response = sendTransaction($args, password)
-  result = response
+proc sendTransaction*(from_address: string, to: string, assetAddress: string, value: string, password: string): string =
+  try:
+    var args: JsonNode
+    if (assetAddress == ZERO_ADDRESS or assetAddress == ""):
+      var weiValue = value.parseFloat() * float(1000000000000000000)
+      var stringValue =  fmt"{weiValue:<.0f}"
+      stringValue.removeSuffix('.')
+      var hexValue = parseBiggestInt(stringValue).toHex()
+      hexValue.removePrefix('0')
+      args = %* {
+        "value": fmt"0x{hexValue}",
+        "from": from_address,
+        "to": to
```

Comment:
is the gas limit estimated automatically?

Reply:
Yes, Status-go does it for us. Later, we'll add a gas price and gas limit fields in the form